### ISSUE TITLE
关于MNER中crfloss的mask问题

### ISSUE THREAD

**Issue by 24kawm** [zjunlp/MKGformer#53]
loss = -1 * self.crf(emissions, labels, mask=attention_mask.byte(), reduction='mean')  # 去掉CLS
在这条函数中，似乎mask并没有其作用，后续会报错labels中的-100引发越界问题，请问我是否能在该函数前将labels和mask按元素相乘后传入该函数，同时取消mask的传入。

**Comment by flow3rdown** [comment]
抱歉，这几天较忙没有及时回复，请您先检查一下是否正确安装了`pytorch-crf`，之后可以尝试提前做一下处理，看一下训练过程有无异常。

**Comment by zxlzr** [comment]
请问您的问题解决了吗

**Comment by 24kawm** [comment]
pytorch-crf版本为0.7.2，提前处理是指将mask和labels提前进行相乘吗，如果是这样的话是可以进行训练的，但是我目前设备太旧无法完成完整的训练，完整训练可能要之后才能实现。
